src/main/routes/api/requests.ts

```diff
@@ -0,0 +1,54 @@
+import { Application } from 'express';
+import * as fs from 'fs';
+import path from 'path';
+
+export default function (app: Application): void {
+  const mocksPath = '../../resources/mocks/';
+
+  app.get('/api/court/:courtId', (req, res) => {
```

Comment:
this is one very long function thats doing multiple things, can we split it so its single responsibility with multiple method calls like `getCourts` or something?